src/main/resources/assets/mekanism/lang/zh_cn.lang

```diff
@@ -100,7 +100,7 @@ tile.BasicBlock.DynamicValve.name=动态阀门
 tile.BasicBlock.CopperBlock.name=铜块
 tile.BasicBlock.TinBlock.name=锡块
 tile.BasicBlock.ThermalEvaporationController.name=热力蒸馏控制器
-tile.BasicBlock.ThermalEvaporationValve.name==热力蒸馏阀门
+tile.BasicBlock.ThermalEvaporationValve.name=热力蒸馏阀门
```

Comment:
already fixed, please revert commit to avoid merge issues later on

Reply:
No, there is no fix here 
Neither the ports of [1.12.2](https://github.com/Thorfusion/Mekanism-Community-Edition/blob/7218280455349aa15486a990fdfc3ebf389dda81/src/main/resources/assets/mekanism/lang/zh_cn.lang#L103) nor [1.12.2-ultimate](https://github.com/Thorfusion/Mekanism-Community-Edition/blob/7218280455349aa15486a990fdfc3ebf389dda81/src/main/resources/assets/mekanism/lang/zh_cn.lang#L103) are fixed
What has been fixed is the port for [1.7.10](https://github.com/Thorfusion/Mekanism-Community-Edition/blob/6fd87104ee69e534bc5032a00d7bf78ad1352751/src/main/resources/assets/mekanism/lang/zh_CN.lang#L100)

Reply:
hahahahaha, that = sign stayed in mek for years without getting fixed, amazing. well then, thx for fix mate!